libtransact/Cargo.toml

```diff
@@ -40,7 +40,9 @@ protoc-rust = "2"
 [features]
 default = []
 nightly = []
-experimental = ["redis-db"]
+experimental = ["redis-db",
+                "key-value-state"]
 sawtooth-compat = ["sawtooth-sdk"]
 ursa-compat = ["ursa"]
 redis-db = ["redis"]
+key-value-state = []
```

Comment:
key-value-state is a different feature than the addressers. Comments below on specific suggestions.